src/foam/nanos/script/Script.js

```diff
@@ -10,7 +10,12 @@ foam.CLASS({
 
   implements: [ 'foam.nanos.auth.EnabledAware' ],
 
-  imports: [ 'scriptDAO' ],
+  requires: [
```

Comment:
Use [trailing commas](https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma) as per the style guide.

Reply:
No, do not use trailing commas, as per the style guide (last line):

https://github.com/foam-framework/foam2/blob/master/doc/guides/StyleGuide.md